client/securedrop_client/resources/css/sdclient.css

```diff
@@ -115,6 +115,39 @@ QWidget {
     background-color: #85f6fe;
 }
 
+#InnerTopPane {
+    background-color: #f3f5f9;
+}
+
```

Comment:
These colours aren't right, someone please help me, this is a cry for help :)  (I will try and make some changes to the button disabled state and the active states to make them less jarring, but help slash a reminder that I don't do a good job at visual things)